README.md

```diff
@@ -1,145 +1,31 @@
-# NSC CSC Website
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
 
-<div id="top"></div>
+# North Seattle College Computer Science Club Website
 
-<!-- PROJECT LOGO -->
-<br />
-<div align="center">
-<h3 align="center">NSC CSC Website</h3>
+This is the website for the Computer Science Club at North Seattle. Besides the website this project exists as a place where people new to Github, Computer Science, and building applications can learn.
 
-  <p align="center">
-    <description of project?>
-    <br />
-    <a href="GITHUB LINK"><strong>Explore the docs »</strong></a>
-    <br />
-    <br />
-    <a href="GITHUB LINK">View Demo</a>
-    ·
-    <a href="GITHUB LINK/issues">Report Bug</a>
-    ·
-    <a href="GITHUB LINK/issues">Request Feature</a>
-  </p>
-</div>
+This project is built with React, ChakraUI, and Node.js.
 
-<!-- TABLE OF CONTENTS -->
-<details>
-  <summary>Table of Contents</summary>
-  <ol>
-    <li>
-      <a href="#about-the-project">About The Project</a>
-      <ul>
-        <li><a href="#built-with">Built With</a></li>
-      </ul>
-    </li>
-    <li>
-      <a href="#getting-started">Getting Started</a>
-      <ul>
-        <li><a href="#prerequisites">Prerequisites</a></li>
-        <li><a href="#installation">Installation</a></li>
-      </ul>
-    </li>
-    <li><a href="#usage">Usage</a></li>
-    <li><a href="#roadmap">Roadmap</a></li>
-    <li><a href="#contributing">Contributing</a></li>
-    <li><a href="#license">License</a></li>
-    <li><a href="#contact">Contact</a></li>
-    <li><a href="#acknowledgments">Acknowledgments</a></li>
-  </ol>
-</details>
-
-<!-- ABOUT THE PROJECT -->
-
-## About The Project
-
-Description of project
-
-<p align="right">(<a href="#top">back to top</a>)</p>
-
-### Built With
-
--   [React](https://reactjs.org/)
--   [Bootstrap](https://getbootstrap.com)
--   [NodeJS](https://nodejs.org/en)
-
-<p align="right">(<a href="#top">back to top</a>)</p>
-
-### Prerequisites
-
-### Installation
-
-1. Contact christopher.hancock@seattlecolleges.edu for project secret keys
-2. Create and navigate into a directory
-    ```sh
-     mkdir club_website
-     cd club_website
-    ```
-3. Clone the repo
-    ```sh
-     git clone https://github.com/NSC-Computer-Science-Club/club_website.git
-    ```
-4. Navigate into the root directory
-
-```sh
-  cd club_website
-```
-
-6. Install required packages
-    ```sh
-     npm install
-    ```
-7. Start server
-    ```sh
-     npm start
-    ```
-
-<p align="right">(<a href="#top">back to top</a>)</p>
-
-<!-- ROADMAP -->
-
-## Roadmap
-
-See the [open issues]("GITHUB LINK"/issues) for a full list of proposed features (and known issues).
-
-<p align="right">(<a href="#top">back to top</a>)</p>
-
-<!-- CONTRIBUTING -->
+## How to Log a Bug, Ask a Question, and/or Request a Feature
+- Navigate to `Issues`
+- click the `New Issue` button
```

Comment:
Typo

Reply:
Can you be specific? I see some capitalization issues I just want to be sure we are talking about the same thing

Reply:
Capitalization issues. I didn't want to spend too much time on it because it isn't super important.